src/point.h

```diff
@@ -168,6 +169,10 @@ struct tripoint {
     constexpr tripoint operator/( const int rhs ) const {
         return tripoint( x / rhs, y / rhs, z / rhs );
     }
+    // rounds values after division
+    constexpr tripoint operator/( const float rhs ) const {
+        return tripoint( std::round( x / rhs ), std::round( y / rhs ), std::round( z / rhs ) );
+    }
```

Comment:
Coordinates scaling code all uses `divide_round_to_minus_infinity`, so absent a compelling reason otherwise I would use that. You'd have to add float overloads of it.

That avoids the case of eg. 1,1,1 / .75 == 1,1,1. -1,-1,-1 / .75 would turn back to -1,-1,-1 though. But still, consistency.

Reply:
Tbh I added this mostly for readability in npc::avoid_friendly_fire (npcmove.cpp:3160), because with typed points you can't operate on the coordinates directly and clang-tidy complained to me about doing it on ints. The use case there is computing the average of some points.

I'm fine with doing that in a dedicated method instead of an operator if that's preferred. Always rounding down would however slightly change the current behavior, or possibly not so slightly in regards to the z-level outcome.

Reply:
Use a custom method for it imo. The operator should behave the same as coordinate scaling cause that feels like principle of least surprise.